cmd/lncli/commands_test.go

```diff
@@ -68,3 +71,64 @@ func TestParseChanPoint(t *testing.T) {
 		}
 	}
 }
+
+// TestParseTimeLockDelta tests parseTimeLockDelta with various
+// valid and invalid input values and verifies the output.
+func TestParseTimeLockDelta(t *testing.T) {
+	t.Parallel()
+	testCases := []struct {
+		timeLockDeltaStr      string
+		expectedTimeLockDelta uint16
+		expectErr             bool
+		expectedErrContent    string
+	}{
+		{
+			"-1",
+			0,
+			true,
+			fmt.Sprintf("fail to parse time_lock_delta: %d "+
+				"to uint64", -1),
+		},
+		{
+			"0",
+			0,
+			false,
+			"",
+		},
+		{
+			"3",
+			3,
+			false,
+			"",
+		},
+		{
+			"65535",
+			65535,
+			false,
+			"",
+		},
+		{
+			"65536",
+			0,
+			true,
+			fmt.Sprintf("time_lock_delta is too big, "+
+				"max value is %d", lnd.MaxTimeLockDelta),
+		},
+		{
+			"18446744073709551616",
+			0,
+			true,
+			fmt.Sprint("fail to parse time_lock_delta:" +
```

Comment:
nit: no space between `+` and string, also can add the string space on this line for consistency

Reply:
This value: `18446744073709551616` is bigger than `math.MaxUint64`, so I just write a string here.